assets/charts/components/inspektor-gadget/values.yaml

```
@@ -0,0 +1,42 @@
+# Default values for inspektor-gadget.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: docker.io/kinvolk/gadget
```

`docker.io` is default ?
```suggestion
  repository: kinvolk/gadget
```